.github/workflows/tests.yml

```diff
@@ -2,7 +2,7 @@ name: Tests
 
 on:
   push:
-    branches: [master]
+    branches: ['*']
```

Comment:
A standard reminder for testing changes)

Reply:
Revert the testing code and I'll approve the PR

Reply:
Finished testing and reverted workflows.